pkg/node/manager/manager.go

```diff
@@ -386,3 +390,22 @@ func canAttachTrunk(node *v1.Node) bool {
 	_, ok := node.Labels[config.HasTrunkAttachedLabel]
 	return ok
 }
+
+// Update nodes on configmap changes
+func (m *manager) UpdateNodesOnConfigMapChanges() error {
+	nodeList, err := m.wrapper.K8sAPI.ListNodes()
```

Comment:
This API doesn't really fit for `NodeManager`. If you look at the other APIs they are all generic but this seems resource specific.

Reply:
It would be good to keep this in ConfigMap reconciler.

Reply:
Good point, I have moved this to the ConfigMap controller.